### PR TITLE
TC_05_19_01 | Verify that deleted document has been moved to the trash by using ACTIONS_options dropdown menu and then deleted permanently

### PR DESCRIPTION
**Pull request by nadya0101**
add test TC_05_19_01

### CODE REVIEW

File: tests/05DocumentsTyped.spec.js [hunk 1]
```diff
@@ -139,7 +141,7 @@ test.describe('DocumentsType', () => {
         await documentsPage.sideMenuDocuments.clickAwaitingSignature();
         await documentsAwaitingPage.table.clickFirstOptionsBtn();
         await documentsAwaitingPage.table.clickSendReminderBtn();
-
+        await deleteModal.clickYesDeleteBtn();
```
Comment (IrynaKolh) [inline tests/05DocumentsTyped.spec.js:144]: удалите эту строчку, она попала случайно при мердже

File: tests/05DocumentsTyped.spec.js [hunk 2]
```diff
@@ -115,7 +117,7 @@ test.describe('DocumentsType', () => {
         });
     });
 
-    test('TC_05_16_01 | Verify the user receives an email reminder to sign the document', async ({
+    test.skip('TC_05_16_01 | Verify that the user receives an email reminder to sign the document', async ({
```
Comment (IrynaKolh) [inline tests/05DocumentsTyped.spec.js:120]: unskip test

File: tests/05DocumentsTyped.spec.js (hunk 1 above)
Comment (nadya0101) [reply tests/05DocumentsTyped.spec.js]: done

File: tests/05DocumentsTyped.spec.js (hunk 2 above)
Comment (nadya0101) [reply tests/05DocumentsTyped.spec.js]: done

**Review by ArinaJur — APPROVED** [review]
LGTM

**Pull request merged**